### PR TITLE
Apply rustfmt to the whole codebase

### PR DESCRIPTION
**Pull request by schneiderfelipe**
I understand this is very opinionated, but it is so common in the Rust ecosystem that I couldn't help myself 😆.
If that's OK, I could add a CI job to check it as well, just let me know.

### CODE REVIEW

**Comment by Luthaf** [comment]
So not using rustfmt was very intentional: every time I tried it, I really disliked the output. It seems to be a bit better now, my only remaining grip is how it formats very similar lines differently if one of them goes above the length limit (see [here](https://github.com/chemfiles/chemfiles.rs/pull/35/files#diff-6ff19a1ff611f3835b5a025663cf4dec901ed914626f78847414836439bd1a79R209-R229) or [here](https://github.com/chemfiles/chemfiles.rs/pull/35/files#diff-6ff19a1ff611f3835b5a025663cf4dec901ed914626f78847414836439bd1a79R209-R229)). I would like if it were to use the same strategy as black does, taking hints from the presence/absence of comma after the last argument to function cal to split it/keep it on a single line. Do you know if there is an option to do that?

Even if there is no such option, I'm willing to give it rustfmt a go in this project and see how it goes!

> I could add a CI job to check it as well, just let me know.

Please do!

**Comment by schneiderfelipe** [comment]
> I would like if it were to use the same strategy as black does, taking hints from the presence/absence of comma after the last argument to function cal to split it/keep it on a single line. Do you know if there is an option to do that?

No idea. It would be a nice thing though, agreed!

File: src/atom.rs [hunk 1]
```diff
@@ -1,7 +1,9 @@
 // Chemfiles, a modern library for chemistry file reading and writing
 // Copyright (C) 2015-2018 Guillaume Fraux -- BSD licensed
-use std::marker::PhantomData;
-use std::ops::{Deref, DerefMut, Drop};
+use std::{
+    marker::PhantomData,
+    ops::{Deref, DerefMut, Drop},
+};
```
Comment (Luthaf) [inline src/atom.rs:6]: Are these rustfmt changes? I did not have them when running locally, and I find they make the code less readable.

File: src/frame.rs [hunk 2]
```diff
@@ -5,9 +5,7 @@ use errors::{check, check_not_null, check_success, Error};
 use property::{PropertiesIter, Property, RawProperty};
 use strings;
 
-use super::{Atom, AtomMut, AtomRef};
-use super::{BondOrder, Residue, Topology, TopologyRef};
-use super::{UnitCell, UnitCellMut, UnitCellRef};
+use super::{Atom, AtomMut, AtomRef, BondOrder, Residue, Topology, TopologyRef, UnitCell, UnitCellMut, UnitCellRef};
```
Comment (Luthaf) [inline src/frame.rs:8]: Same here, the separate imports are intentional

Comment (schneiderfelipe) [reply src/frame.rs:8]: Sure thing, let me revert those changes.

**Comment by schneiderfelipe** [comment]
I made two commits, one enforcing [`imports_granularity`](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#imports_granularity) by [module](https://github.com/chemfiles/chemfiles.rs/pull/35/commits/5a6592009255aa6621f188637a17e3e24aa35206) and the other by [item](https://github.com/chemfiles/chemfiles.rs/pull/35/commits/a645b9abe41feed01a0fd3d0971d0ad023638cc5). I feel *by item* is more like what you want.

See if you prefer one of those. If yes, I can enforce it in `rustfmt.toml`. On the contrary, just let me know and I revert them back to how it was 😎.

**Comment by Luthaf** [comment]
If you are happy with the changes here, I can merge this and we can get to your other PR! Hopefully they should be much faster to merge!

**Comment by schneiderfelipe** [comment]
> If you are happy with the changes here, I can merge this and we can get to your other PR! Hopefully they should be much faster to merge!

Perfect! ✨

**Pull request merged**